### DISCUSSION TITLE
QUESTION - Mod pyLoad WebInterface - How to add a Restart NAS button?

### DISCUSSION THREAD

**Discussion by Ryan747800B** [pyload/pyload#3977] [Help]
How is it possible to add a button - for example, next to "Exit" - so that I can reboot remotely my NAS - a My Book Live - without exposing its Dashboard to Internet? Don't know, maybe linking it to a reboot script, so that I don't need to enable SSH?
I really don't know how can I mod the WebInterface, what files are involved?

**Comment by GammaC0de** [comment]
Not possible in v0.4.9
The web interface has an option to restart pyload though.

**Comment by Nippey** [comment]
Just my few cents: This may be combined with issue/pull-request #1521 ;)
